packages/opentelemetry-exporter-collector/src/platform/browser/CollectorMetricExporter.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { MetricRecord, MetricExporter } from '@opentelemetry/metrics';
+import * as collectorTypes from '../../types';
+import { CollectorExporterBrowserBase } from './CollectorExporterBrowserBase';
+import { toCollectorExportMetricServiceRequest } from '../../transformMetrics';
+import { CollectorExporterConfigBrowser } from './types';
+
+const DEFAULT_COLLECTOR_URL = 'http://localhost:55680/v1/metrics';
+const DEFAULT_SERVICE_NAME = 'collector-metric-exporter';
+
+/**
+ * Collector Metric Exporter for Web
+ */
+export class CollectorMetricExporter
+  extends CollectorExporterBrowserBase<
+    MetricRecord,
+    collectorTypes.opentelemetryProto.collector.metrics.v1.ExportMetricsServiceRequest
+  >
+  implements MetricExporter {
+  private readonly _startTime = new Date().getTime() * 1000000;
```

Comment:
Why are we multiplying by `1_000_000`?

efficiency nit: `Date.now()` prevents the allocation of a new Date object.

Reply:
I did this to convert time to nanoseconds in order to comply with MetricRecord. Is there a preferred way of doing that?

Reply:
No, I just think a comment here "convert to nanoseconds" would go a long way.

Reply:
Ok sounds good, will do